Pdf/PdfDocumentGenerator.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using DocumentService.Pdf.Models;
+using System.Diagnostics;
+
+namespace DocumentService.Pdf
+{
+    public class FilePathDoesNotExist : Exception
+    {
+        public FilePathDoesNotExist(string msg) : base(msg)
+        {
+
+        }
+    }
+     
+    public class PdfDocumentGenerator
+    {
+        public static void generatePdfByTemplate(string templatePath, List<ContentMetaData> metaDataList, string outputFilePath)
```

Comment:
`generatePdfByTemplate` -> `GeneratePdfByTemplate`
The convention followed to name the class methods is Pascal case.

Reply:
Added pascal case to the class name